update EDL_description.md (#31316)
demisto/content: Packs/EDL/ReleaseNotes/3_2_16.md

#### Integrations

##### Generic Export Indicators Service

> - Added the *Listen Port* and *credentials* parameters to XSOAR SAAS.

Reviewer: ```suggestion
- Added back the *Listen Port* and *Username* parameters for XSOAR SaaS.
```
